apps/backoffice-v2/src/pages/Entity/components/Case/hooks/usePendingRevisionEvents/utils/calculate-workflow-revisionable-event.tsx

```diff
@@ -0,0 +1,20 @@
+import { TWorkflowById } from '@/domains/workflows/fetchers';
+import { getPostDecisionEventName } from '@/lib/blocks/components/CallToActionLegacy/hooks/useCallToActionLegacyLogic/useCallToActionLegacyLogic';
+import { CommonWorkflowEvent } from '@ballerine/common';
+
+export const calculateWorkflowRevisionableEvent = (
+  workflow: TWorkflowById,
+  documentStatus: string,
+) => {
+  const postDecisionEvent = getPostDecisionEventName(workflow);
+  const dispatchableEvent =
+    postDecisionEvent || documentStatus === CommonWorkflowEvent.REVISION
+      ? (CommonWorkflowEvent.REVISION as string)
+      : undefined;
```

Comment:
Consider if statements over ternaries for similar logic. It does not seem the variable is in use beyond returning it.